.github/workflows/api-compatibility.yml

```
@@ -0,0 +1,127 @@
+name: API Compatibility (Valkey.Glide vs StackExchange.Redis)
+
+on:
+    pull_request:
+        branches: [main]
+    workflow_dispatch:
+        inputs:
+            ser_version:
+                description: StackExchange.Redis version to compare against
+                required: false
+                default: "2.8.58"
+
+permissions:
+    contents: read
+
+jobs:
+    api-compatibility:
+        runs-on: windows-latest
+        env:
+            # Default for PRs; workflow_dispatch can override via input
+            SER_VERSION: ${{ github.event_name == 'workflow_dispatch' && inputs.ser_version || '2.8.58' }}
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  submodules: true
+
+            - name: Setup .NET
+              uses: actions/setup-dotnet@v4
+              with:
+                  dotnet-version: |
+                      8.x
+
+            # Install Rust toolchain on Windows for cargo prebuild
+            # - name: Install Rust (Windows)
+            #   shell: pwsh
+            #   run: |
+            #       $ProgressPreference = 'SilentlyContinue'
+            #       Invoke-WebRequest -Uri https://win.rustup.rs/x86_64 -OutFile rustup-init.exe
+            #       ./rustup-init.exe -y
+            #       echo "$env:USERPROFILE\.cargo\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+            #       rustc -V
+            #       cargo -V
+
+            - name: Install dotnet-api-diff (Chocolatey)
+              shell: pwsh
+              run: |
+                  choco install dotnetapidiff --version=1.0.0 -y
+                  dotnetapidiff --version
+
+            # - name: Restore dependencies (for tooling)
+            #   shell: pwsh
+            #   run: |
+            #       dotnet restore benchmarks/csharp_benchmark.csproj
+
+            - name: Download StackExchange.Redis package (${{ env.SER_VERSION }})
+              shell: pwsh
+              run: |
+                  nuget install StackExchange.Redis -Version "$env:SER_VERSION" -OutputDirectory ./packages
+
+            - name: Build Valkey.Glide (net8.0)
+              shell: pwsh
+              run: |
+                  dotnet build --configuration Release --framework net8.0 --nologo
```

```suggestion
                  dotnet build --configuration Lint --framework net8.0 --nologo
```
to skip building rust part
(or set `SkipCargo` env var after merging #12)

---

I do not want lint build here. We have separate builds that focus on linting. This only exists to give me the assembly to test.

---

Lint did exactly what I thought it might do, it failed because of linting errors. Reverting back to a release build.